core/src/test/java/nl/nn/adapterframework/xslt/XsltErrorTestBase.java

```diff
@@ -65,10 +65,11 @@ public boolean isEmpty() {
 	@Before
 	public void init() {
 		// Force reconfigure to clean list appender.
-//		Configurator.reconfigure();
-		testAppender = TestAppender.newBuilder().useIbisPatternLayout("%level %m").build();
+		testAppender = TestAppender.newBuilder()
+			.useIbisPatternLayout("%level %m")
+			.minLogLevel(Level.WARN)
+			.build();
```

Comment:
Handig 💪🏻!
Misschien ook leuk om op log messages van bepaalde packages/classes te kunnen filteren 😃

Reply:
Dat heb ik nu mogelijk gemaakt (door niet toe te voegen aan de root-appender, maar aan een andere appender bij naam) maar dat was voor de XSLT testen alleen niet er nuttig.
Misschien wel nuttig voor andere testen, in de toekomst!